app/src/org/commcare/fragments/connect/ConnectDeliveryProgressDeliveryFragment.java

```diff
@@ -224,11 +231,17 @@ public void updateView() {
         else if(job.getProjectStartDate() != null && job.getProjectStartDate().after(new Date())) {
             //Project hasn't started yet
             text = getString(R.string.connect_progress_begin_date, ConnectManager.formatDate(job.getProjectStartDate()), endText);
-        }
-        else {
+        } else if (job.getIsUserSuspended()) {
+            text = getString(R.string.user_suspended);
+        } else {
             text = getString(R.string.connect_progress_complete_by, endText);
         }
         textView.setText(text);
+        if(!job.getIsUserSuspended()) {
+            textView.setTextColor(getResources().getColor(R.color.red));
```

Comment:
Nit: simplify the code in the case-structure, i.e.:
int color = job.getIsUserSuspended() ? R.color.red : R.color.black;
textView.setTextColor(getResources().getColor(color));